packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -475,10 +473,8 @@ class Splitter extends (CollectionWidget as any) {
   _getResizeHandlesSize(): number {
     let size = 0;
 
-    this._resizeHandles.forEach((resizeHandle) => {
-      const { disabled, separatorSize } = resizeHandle.option();
-
-      size += disabled ? INACTIVE_RESIZE_HANDLE_SIZE : separatorSize as number;
+    this._resizeHandles.forEach((resizeHandle: ResizeHandle) => {
```

Comment:
_getResizeHandlesSize(): number {
  return this._resizeHandles.reduce((totalSize, resizeHandle) => totalSize + resizeHandle.getSize(), 0);
}